dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/data_configurations_panel.tsx

```diff
@@ -219,22 +260,32 @@ export const DataConfigPanelItem = ({
     (visualizations.vis.name === VIS_CHART_TYPES.Line ||
       visualizations.vis.name === VIS_CHART_TYPES.Scatter);
 
+  const getTimeStampFilteredFields = (options: IField[]) =>
+    filter(options, (i) => i.type !== TIMESTAMP);
```

Comment:
See if it is possible to add type to variables.

Reply:
Done